CMAKE.md

```diff
@@ -0,0 +1,31 @@
+# Building with CMake
+
+Some portions of this repository are portable beyond iOS and can be built using
+CMake.
+
+## Dependencies
+
+You need:
+  * `cmake`
+  * `ninja` (optional)
+
+Install with your favorite package manager for your platform, e.g. homebrew on
+macOS, chocolatey on Windows, apt-get on Ubuntu, etc.
+
```

Comment:
Give the `brew install cmake ninja` and other install commands as an example - so people with at least one package manager don't need to remember proper syntax

Reply:
Done. I've given instructions for all the platforms.